### PR TITLE
Add eslint loader

### PR DESCRIPTION
**Pull request by FreaKzero**

### CODE REVIEW

**Comment by karlhorky** [comment]
What happens if we add this to the server-side webpack config too?

File: packages/react-scripts/kununu/config/webpack.client.config.js [hunk 1]
```diff
@@ -6,12 +6,13 @@ const patchResolve = require('../utils/webpack/patchResolve');
...
 const patchSassLoader = require('../utils/webpack/patchSassLoader');
 const patchCssLoader = require('../utils/webpack/patchCssLoader');
+const patchESLintLoader = require('../utils/webpack/patchESLintLoader');
 const disableExtractTextPlugin = require('../utils/webpack/disableExtractTextPlugin');
 
 module.exports = function webpackClientConfig (webpackConfig) {
   // This check is not really correct because we actually needed to check for the hot-option. Unfortunately, we can't access
   // these configuration values here, so let's assume that NODE_ENV = "development" roughly equals the hot-option.
-  const hotIsEnabled = process.env.NODE_ENV === 'development';
+  const isDevEnvironment = process.env.NODE_ENV === 'development';
```
Comment (karlhorky) [inline packages/react-scripts/kununu/config/webpack.client.config.js]: Reading the comment above, we may want to keep this naming for the hot variable.

Maybe we want to reassign `isDevEnvironment` to a second variable here (for `hotIsEnabled`) - for use with the `disableExtractTextPlugin(webpackConfig)` below.

**Comment by FreaKzero** [comment]
> What happens if we add this to the server-side webpack config too?
We have no sync errors so the question would be if we really need that right now ?

File: packages/react-scripts/kununu/config/webpack.client.config.js (hunk 1 above)
Comment (FreaKzero) [inline packages/react-scripts/kununu/config/webpack.client.config.js]: Lets wait until this issue which is in the comments gets resolved (so it uses an actual hotloader check) and then we should introduce an `isDevEnvironment` constant.

**Comment by karlhorky** [comment]
Sounds good, let's leave this out for now and add it if we need it later.

**Pull request merged**